src/sonic-eventd/Makefile

```diff
@@ -29,7 +29,7 @@ endif
 -include rsyslog_plugin/subdir.mk
 -include rsyslog_plugin_tests/subdir.mk
 
-all: sonic-eventd eventd-tests eventd-tool rsyslog-plugin rsyslog-plugin-tests
+all: sonic-eventd eventd-tool rsyslog-plugin
```

Comment:
it is that a mistake to disable rsyslog-plugin-tests also?

Reply:
Hi @StormLiangMS  We have build failures when compiling sonic-eventd target and I disabled all unit tests under this target.

Reply:
Technically, there is no issue with rsyslog-plugin tests only with eventd-tests.

Reply:
@zbud-msft pls let me know when your fix is ready.